helium/build.gradle

```diff
@@ -22,4 +22,5 @@ dependencies {
     exclude group: 'org.codehaus.groovy'
   }
   testCompile 'com.google.mockwebserver:mockwebserver:20130403'
+  testCompile 'com.squareup.okhttp:mockwebserver:2.3.0'
```

Comment:
We should probably remove old `com.google.mockwebserver`. Will you afford doing this in this PR?